packages/flutter/lib/src/material/input.dart

```diff
@@ -135,52 +144,30 @@ class _InputFieldState extends State<InputField> {
   }
 }
 
-/// A material design text input field.
-///
-/// Requires one of its ancestors to be a [Material] widget.
+/// Displays the visual elements of a material design text field. This
+/// widget's child is the input field widget itself.
```

Comment:
maybe s/text/form/ here, so people think they can use this for things other than text fields?